lib/karaokium/performances.ex

```diff
@@ -46,6 +46,9 @@ defmodule Karaokium.Performances do
   """
   def get_performance!(id), do: Repo.get!(Performance, id)
```

Comment:
```suggestion
```

Reply:
I just changed the name of other function ,which had the same name but has different parameters. I did this because this function was already called some times, so in order to not change many things i just change the name of the new function that I created. The name was "get_performance" and now is "get_performance2"

Reply:
The thing is, they are the same function, which means you can only delete the first one and everything will still work.

When you call `get_performance!(id)` without the second argument, it will have by default an empty list (`[]`). Because the signature of the second one is `def get_performance(id, preloads \\ [])` where `\\ []` means default value.

Learn more about default params [here](https://elixir-lang.org/getting-started/modules-and-functions.html#default-arguments).

Reply:
No need to apologize. It amazing work this PR 💪